content/handbook-clustered.md

```diff
@@ -0,0 +1 @@
++++ template = "handbook-clustered.html" +++
```

Comment:
Github's default view of this template is slightly confusing. There need to be line breaks for each line as seen here: https://github.com/laniakeabooks/laniakeabooks.org/blob/main/content/how-it-works.md?plain=1